test/functional/autocmd/tabnewentered_spec.lua

```diff
@@ -29,3 +37,351 @@ describe('TabNewEntered', function()
     end)
   end)
 end)
+
+describe('tabpage/earlier', function()
```

Comment:
"previous" is the Vim conventional name for this concept, so would be nice to use that jargon here and below rather than "earlier".

Reply:
Fixed.  The only trouble here is `help :tabprevious`.